contracts/interfaces/IWitnetRNG.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "../libs/Witnet.sol";
+
+/// @title The Witnet Random Number Generation provider interface.
+/// @author The Witnet Foundation.
+interface IWitnetRNG {
```

Comment:
From a marketing perspective, we'd better forget about RNG and focus on _randomness_. This could be `IWitnetRandom` or `IWitnetRandomness`

Reply:
For comments on interface docs, see comments on implementation.